services/slack/intermediate.go

```diff
@@ -392,6 +393,10 @@ func TransformPosts(slackExport *SlackExport, intermediate *Intermediate, attach
 					}
 				}
 
+				if len(post.Attachments) > 0 {
```

Comment:
Would we like to add the fallback to all the posts that contain attachments or only to those that are marked as "shared" somehow?

Reply:
As far as I could tell from the Slack documentation the `attachments` key apparently is always meant to carry additional text for a post that's being posted in a *quote* format.

See here: https://api.slack.com/messaging/composing/layouts#attachments

Feel free to correct me if I got this wrong.